src/mscorlib/src/System/IO/Directory.cs

```diff
@@ -673,20 +673,21 @@ public static String[] GetLogicalDrives()
         public static String GetCurrentDirectory()
         {
             // Start with a buffer the size of MAX_PATH
-            using (StringBuffer buffer = new StringBuffer(260))
+            StringBuffer buffer = new StringBuffer(260);
```

Comment:
Because of you cannot pass `using` variables via `ref`. It would be really nice to have better support for the `MustNotCopy` style in C#... .